docs/docs/quickstart.mdx

```diff
@@ -117,52 +116,25 @@ app.use(pathPrefix + '/', (req, res) => {
 // ...
 ```
 
-ORY Kratos does not ship with an administrative user interface. You must
+Kratos does not ship with an administrative user interface. You must
 implement that yourself or choose the ORY Cloud offering (to be announced). In
-this quickstart, we will use ORY Kratos CLI (Command Line Interface) to interact
-with ORY Kratos' Administrative APIs.
+this quickstart, we will use Kratos CLI (Command Line Interface) to interact
+with Kratos' Administrative APIs.
 
 The quickstart also comes with [MailSlurper](https://mailslurper.com), a mock
 SMTP server the demo uses to show how e.g. email verification works.
 
-### Clone ORY Kratos and run it in Docker
-
-To get this example working, you will need Git and
-[Docker, and Docker Compose](https://docs.docker.com/get-docker/) installed on
-your system. No other dependencies are required. Before you start, make sure
-that Docker has enough disk space.
+### Clone Kratos and run it in Docker
 
-:::tip
+To get this example working, you will need Git,
+[Docker](https://docs.docker.com/get-docker/), and [Docker Compose](https://docs.docker.com/compose/install/) installed on
+your system. No other dependencies are required.
 
-This tutorial uses Docker-Compose volumes which have reported to run out of disk
-space. Check the remaining disk space using `docker system df`. If the volumes
-are above the 85% threshold,
-[prune old Docker objects](https://docs.docker.com/config/pruning/) before you
-start!
-
-:::
-
-:::tip
-
-If you encounter build errors (e.g. network timeout), make sure that the network
-is running correctly and run `macke docker` again. If the problem persists, feel
-free to [open an issue](https://github.com/ory/kratos/issues/new/choose).
-
-:::
-
-Let's clone ORY Kratos and run `docker-compose`:
+Let's clone Kratos and run `docker-compose`:
 
 ```shell script
-git clone https://github.com/ory/kratos.git
-# or if you have git+ssh set up:
-#  git clone git@github.com:ory/kratos.git
-cd kratos
-git checkout v0.3.0-alpha.1
-make quickstart
-
-# or if you don't have make installed:
 docker pull oryd/kratos:latest-sqlite
-docker pull oryd/kratos-selfservice-ui-node:latest
+docker pull oryd/kratos-selfservice-ui-node
 docker-compose -f quickstart.yml -f quickstart-standalone.yml up --build --force-recreate
 ```
```

Comment:
Please revert these changes, otherwise the chat will be flooded again with people complaining about Docker not working.

Reply:
Ah, I was under the impression if you don't specify a tag that it was the same thing as specifying `latest`.